com.unity.perception/Runtime/GroundTruth/SimulationState.cs

```diff
@@ -674,7 +667,7 @@ public AsyncAnnotation ReportAnnotationAsync(AnnotationDefinition annotationDefi
             foreach (var value in values)
                 jArray.Add(new JRaw(DatasetJsonUtility.ToJToken(value)));
 
-            ReportAsyncAnnotationResult<T>(asyncAnnotation, filename, jArray);
+            ReportAsyncAnnotationResult(asyncAnnotation, filename, jArray);
         }
 
         [SuppressMessage("ReSharper", "PossibleInvalidOperationException")]
```

Comment:
It looks like QuaternionConvertor and Vector3Convertor are no longer needed. The only other usage of them is in tests. These were used for creating a specific output format but it later the code was changed to just use unity's native serialization.